shared/actions/platform-specific/index.native.tsx

```diff
@@ -878,6 +836,20 @@ const onSendAudioRecording = (action: Chat2Gen.SendAudioRecordingPayload) => {
   }
 }
 
+const onTabLongPress = (state: Container.TypedState, action: RouteTreeGen.TabLongPressPayload) => {
```

Comment:
new action to handled long presses. only supporting quick switching users using the people tab. this logic moved out of the actual tab button itself